src/pages/aurora-site/aurora-chat/index.jsx

```
@@ -1,119 +1,142 @@
 const AuroraChat = () => {
   const [messages, setMessages] = useState([
     {
       type: "text",
-      content: "Hello! I'm your AURORA AI language assistant. How can I help you with your language learning today?",
+      content:
+        "Hello! I'm your AURORA AI language assistant. How can I help you with your language learning today?",
       isEliza: true,
-    }
+    },
   ]);
   const [inputText, setInputText] = useState("");
```

ESLint

[error] 'setInputText' is assigned a value but never used.

(no-unused-vars)